app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+
+from dotenv import load_dotenv
+
+import requests
+
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("API_KEY environment variable is not set")
+
+    city = "Paris"
```

Comment:
The variable `city` should be defined as a constant in uppercase, e.g., `CITY`, to follow the convention for constants.